modules/httpcache/httpcache.go

```diff
@@ -19,13 +19,13 @@ import (
 // AddCacheControlToHeader adds suitable cache-control headers to response
 func AddCacheControlToHeader(h http.Header, d time.Duration) {
 	if setting.IsProd {
-		h.Set("Cache-Control", "private, max-age="+strconv.Itoa(int(d.Seconds())))
+		h.Set("Cache-Control", "private, no-transform, max-age="+strconv.Itoa(int(d.Seconds())))
 	} else {
-		h.Set("Cache-Control", "no-store")
+		h.Set("Cache-Control", "no-store, no-transform")
 		// to remind users they are using non-prod setting.
 		// some users may be confused by "Cache-Control: no-store" in their setup if they did wrong to `RUN_MODE` in `app.ini`.
 		h.Add("X-Gitea-Debug", "RUN_MODE="+setting.RunMode)
-		h.Add("X-Gitea-Debug", "CacheControl=no-store")
+		h.Add("X-Gitea-Debug", "CacheControl=no-store, no-transform")
```

Comment:
Maybe it's time to remove this `h.Add("X-Gitea-Debug", "CacheControl`, the `"X-Gitea-Debug" with RUN_MODE` should be clear enough to remind users about their RUN_MODE. I wrote the code for some old issues, there is no such issue anymore.

Reply:
Already did that actually.